### PR TITLE
Add rebars to Group Python feature object

### PR DESCRIPTION
**Pull request by SurajDadral**
Commits:
- Add rebars to Group Python feature object
- Open editDialog by double click group in tree view
- Set values in edit dialog box
- Modify code structure

### CODE REVIEW

File: ColumnReinforcement/SingleTie.py [hunk 1]
```diff
@@ -283,20 +294,23 @@ def makeSingleTieFourRebars(
         b_cover = tb_cover[1]
 
         i = 0
+        main_rebars = []
         for orientation in list_orientation:
```
Comment (amrit3701) [inline ColumnReinforcement/SingleTie.py]: Used `enumerate` instead to use `i = 0`.

```
for i, orientation in enumerate(list_orientation)
```

File: ColumnReinforcement/SingleTie.py [hunk 2]
```diff
@@ -307,18 +321,142 @@ def makeSingleTieFourRebars(
...
+    'LShapeRebar'.
+    It takes eight different orientations input for L-shaped hooks i.e. 'Top
+    Inside', 'Top Outside', 'Bottom Inside', 'Bottom Outside', 'Top Left',
+    'Top Right', 'Bottom Left', 'Bottom Right'.
+    It takes two different inputs for hook_extend_along i.e. 'x-axis', 'y-axis'.
+    """
+    print("Implementation in progress")
+
+
+class _SingleTieFourRebars(_RebarGroup, _ViewProviderRebarGroup):
```
Comment (amrit3701) [inline ColumnReinforcement/SingleTie.py]: No need to inherit `_SingleTieFourRebars` from `_ViewProviderRebarGroup`.

**Comment by amrit3701** [comment]
Also, your many lines exceed more than 80 columns width. Please fix it.

**Comment by SurajDadral** [comment]
Done above suggested changes in latest branch [gsoc19-ui](https://github.com/SurajDadral/FreeCAD-Reinforcement/tree/gsoc19-ui)
Commit: https://github.com/SurajDadral/FreeCAD-Reinforcement/commit/fb6c23d6919db55892e50ce12e1fbd200b9a0e35
PR: https://github.com/amrit3701/FreeCAD-Reinforcement/pull/22

**Pull request merged**